src/catkin_pkg/package_templates.py

```diff
@@ -309,7 +311,7 @@ def _create_include_macro(package_template):
         if deplist:
             result += '\n# TODO: Check names of system library include directories (%s)' % ', '.join(deplist)
```

Comment:
This message will now appear above all the includes and not anymore directly above `deplist`. Maybe just append this comment to `includes` (without the leading newline)?